src/syscheckd/create_db.c

```diff
@@ -147,7 +147,8 @@ void fim_checker(char *path, fim_element *item, whodata_evt *w_evt, int report)
 
         if (saved_entry) {
             json_event = fim_json_event(path, NULL, saved_entry->data, item->index, FIM_DELETE, item->mode, w_evt);
-            fim_db_remove_path(syscheck.database, saved_entry, &syscheck.fim_entry_mutex, (void *) (int) 0);
+            fim_db_remove_path(syscheck.database, saved_entry, &syscheck.fim_entry_mutex, (void *) (int) false,
```

Comment:
To make it clearer that it is not sending an alert.
That value is only checked here:  

https://github.com/wazuh/wazuh/blob/b1b4d075e27bf6e7a8227957f5d3c52e1b9e2e7c/src/syscheckd/fim_db.c#L1110

I thought it would be clearer since it works as a boolean.